src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Drawable, AreaCalculable {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super.setColor(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {
+        return Math.PI * Math.pow(radius,2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area:" + area() + "sq. units, radius: "
```

Comment:
Use only eng in messages/code: The message in the 'System.out.println()' should be in English. Also, ensure there is a space after the colon following 'area' for proper formatting.